src/marginalkde.jl

```diff
@@ -0,0 +1,73 @@
+@userplot MarginalKDE
+
+@recipe function f(kc::MarginalKDE; levels=10, clip=((-3.0, 3.0), (-3.0, 3.0)))
+    x,y = kc.args
+
+    x = vec(x)
+    y = vec(y)
+
+    m_x = median(x)
+    m_y = median(y)
+
+    dx_l = m_x - quantile(x, 0.16)
+    dx_h = quantile(x, 0.84) - m_x
```

Comment:
Are 0.16 and 0.84 the best choice in all cases, or should we add an additional kwarg or alternatively specify `xlims` and `ylims` using `-->` in the recipe?

Reply:
Hmm.  That's controllable via the `clip` argument (which multiplies these values); those numbers would represent +/- 1-sigma in a Gaussian, so are probably sensible for choosing a "scale" that gets multiplied by the `clip` argument.

Reply:
Oh, sorry, I did not take a close enough look.